src/blockchain_db/lmdb/db_lmdb.h

```diff
@@ -39,15 +39,20 @@
 namespace cryptonote
 {
 
+typedef struct mdb_block_info
+{
+  uint64_t bi_timestamp;
+  uint64_t bi_coins;
+  size_t bi_size;
+  difficulty_type bi_diff;
```

Comment:
difficulty_type -- difficulty.h:    typedef std::uint64_t difficulty_type;
Shouldn't that already be the same as uint64_t?